src/styles/utils/variables.scss

```diff
@@ -0,0 +1,18 @@
+$header-color: #00acdc;
+$color-text: #060b35;
+$color-text-small: #616070;
+$color-button: #00acdc;
+$color-button-second: #fff;
+$color-title-hover: #34568b;
+$grid-colums-1: 1;
+$grid-colums-2: 2;
+$grid-colums-3: 3;
+$grid-colums-4: 4;
```

Comment:
It seems there is a small typo in the variable names for grid columns. The standard convention for naming variables is to use hyphens rather than underscores to separate words. Consider renaming `$grid-colums-1` to `$grid-columns-1`, and similarly for the other grid column variables.